examples/with-redis/postcss.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   plugins: {
-    '@tailwindcss/jit': {},
+    tailwindcss: {},
     autoprefixer: {},
   },
-}
+}
```

Comment:
Adding a new line should fix the issue.

> Code style issues found in the above file(s). Forgot to run Prettier?

```suggestion
}

```